jstreegrid.js

```diff
@@ -41,11 +41,13 @@
     getIds = function (nodes) {
       return $.makeArray(nodes.map(function(){return this.id;}));
     },
-		findDataCell = function (uniq, ids, col) {
-			var ret = $(), columns = [].concat(col), cellId;
+    findDataCell = function (uniq, ids, col, scope) {
+      if (scope == undefined) { scope = $(); };
 			if (ids === null || ids === undefined || ids.length === 0) {
-				ret = "";
-			} else if (typeof(ids) === "string") {
+        return scope;
```

Comment:
Returning an empty string was an error, but this is just a fallback scenario, cause ids is always provided